core/src/main/java/io/grpc/transport/AbstractStream.java

```diff
@@ -295,7 +295,7 @@ protected final boolean isDeframerStalled() {
    * Called to request the given number of messages from the deframer. Must be called
    * from the transport thread.
    */
-  protected final void requestMessagesFromDeframer(int numMessages) {
+  public final void requestMessagesFromDeframer(int numMessages) {
```

Comment:
This should remain protected and it can be re-defined public (or even protected, because of package-private) in Netty's versions.